templates/instantanswer/new_ia.tx

```diff
@@ -154,3 +135,6 @@
         </ul>       
     </div>
 </div>
+<: if $c.session.ia_data { :>
+    <div class="hide" id="session_ia_data"></div>
+<: } :>
```

Comment:
do you prefer to use the inline syntax everywhere instead of the xslate line-oriented syntax?

```
: if $c.session.ia_data {
     <div class="hide" id="session_ia_data"></div>
: }
```